cirq-core/cirq/sim/sparse_simulator.py

```diff
@@ -296,8 +310,14 @@ def state_vector(self, copy: bool = True):
                 parameters from the state vector and store then using False
                 can speed up simulation by eliminating a memory copy.
         """
-        vector = self._simulator_state().state_vector
-        return vector.copy() if copy else vector
+        if self._state_vector is None:
+            self._state_vector = np.array([1])
+            state = self.merged_sim_state
+            if state is not None:
+                vector = state.target_tensor
+                size = np.prod(vector.shape, dtype=int)
+                self._state_vector = np.reshape(vector, size)
+        return self._state_vector.copy() if copy else self._state_vector
 
     def set_state_vector(self, state: 'cirq.STATE_VECTOR_LIKE'):
```

Comment:
Make sure this is fixed per https://github.com/quantumlib/Cirq/pull/4100#issuecomment-864257447